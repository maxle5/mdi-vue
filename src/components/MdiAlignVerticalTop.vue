
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M22 2V4H2V2H22M7 22H10V6H7V22M14 16H17V6H14V16Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiAlignVerticalTop",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
