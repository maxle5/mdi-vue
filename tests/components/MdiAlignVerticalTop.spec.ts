
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiAlignVerticalTop from "../../src/components/MdiAlignVerticalTop.vue";

test("MdiAlignVerticalTop snapshot", () => {
  const wrapper = mount(MdiAlignVerticalTop, {});
  expect(wrapper.html()).toMatchSnapshot();
});
