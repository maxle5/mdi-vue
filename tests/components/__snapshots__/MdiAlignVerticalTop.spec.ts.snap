// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiAlignVerticalTop snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M22 2V4H2V2H22M7 22H10V6H7V22M14 16H17V6H14V16Z" style="fill: currentColor;"></path>
</svg>"
`;
